app/presenters/ministers_index_presenter.rb

```diff
@@ -118,4 +139,8 @@ def ordered_whip_organisations
   end
 
   WhipOrganisation = Struct.new(:item_key, :name_key, :ministers, keyword_init: true)
+
+  def org_role_ids(org_data)
```

Comment:
Just a really pedantic thing: maybe this could be renamed `role_ids_for_organisation` to be clearer what the IDs actually relate to?